index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 not found</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&family=Merriweather&family=Montserrat&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="main">
+    <div class="not-found">
+      <h1 class="letter-not-found">404 not found</h1>
+    </div>
+  </div>
+  <div class="body">
+    <div class="scarecrow"></div>
+    <div class="bad-news-title">
+      <h1 class="bad-news"> I have bad news for you</h1>
+    </div>
+    <div class="phar-info">
+      <p class="information">The page you are looking for might be removed or is temporarily unavailable</p>
+    </div>
+    <a href="#" class="button">Back to homepage</a>
+  </div>
+  <div class=" footer">
+    <div class="credits">
+      <p class="credit-creator">created by <strong>username</strong>- devChallenges.io</p>
+    </div>
+  </div>
+
+
+</body>
+
+</html>
```

Comment:
A blank line is needed at the bottom of this file to avoid compilation errors in some tools.